pkgs/tools/package-management/lix/common.nix

```diff
@@ -220,15 +226,27 @@ stdenv.mkDerivation {
       done
     '';
 
+  # This needs to run after _multioutDocs moves the docs to $doc
+  postFixup = ''
```

Comment:
```suggestion
  postFixup = lib.optionalString enableDocumentation ''
```

$devdoc and $doc don't exist unless documentation is enabled, so this fails when it's not

Reply:
oh yeah that's busted. i think i fixed it, testing it right now.